Corona-Warn-App/src/main/res/layout/rat_profile_onboarding_fragment.xml

```diff
@@ -1,45 +1,273 @@
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toEndOf="@id/rat_profile_onboarding_bullet_point5"
+                    app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_body4" />
+
+                <ImageView
+                    android:id="@+id/rat_profile_onboarding_bullet_point6"
+                    android:layout_width="8dp"
+                    android:layout_height="8dp"
+                    android:layout_marginTop="22dp"
+                    android:importantForAccessibility="no"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_body5"
+                    app:srcCompat="@drawable/bullet_point" />
+
+                <TextView
+                    android:id="@+id/rat_profile_onboarding_body6"
+                    style="@style/subtitle"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="15dp"
+                    android:layout_marginTop="16dp"
+                    android:focusable="true"
+                    android:text="@string/rat_profile_onboarding_card_consent6"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toEndOf="@id/rat_profile_onboarding_bullet_point6"
+                    app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_body5" />
+
+                <ImageView
+                    android:id="@+id/rat_profile_onboarding_bullet_point7"
+                    android:layout_width="8dp"
+                    android:layout_height="8dp"
+                    android:layout_marginTop="22dp"
+                    android:importantForAccessibility="no"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_body6"
+                    app:srcCompat="@drawable/bullet_point" />
+
+                <TextView
+                    android:id="@+id/rat_profile_onboarding_body7"
+                    style="@style/subtitle"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="15dp"
+                    android:layout_marginTop="16dp"
+                    android:focusable="true"
+                    android:text="@string/rat_profile_onboarding_card_consent7"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toEndOf="@id/rat_profile_onboarding_bullet_point7"
+                    app:layout_constraintTop_toBottomOf="@id/rat_profile_onboarding_body6" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+            <de.rki.coronawarnapp.util.ui.views.MoreInformationView
```

Comment:
clickListener is missing in `RATProfileOnboardingFragment`

Reply:
it is not missing ,in figma it doesn't say it should navigate somewhere and the text looks different from other onboarding screen